### PR TITLE
Provide list of URI-Rs and datetimes. Slight refactoring

### LINKED ISSUE #543: ipwbAdmin: Provide list of URI-Rs and datetimes available

### PR DESCRIPTION
**Pull request by machawk1**
Closes #543

### CODE REVIEW

File: ipwb/assets/admin.css [hunk 1]
```diff
@@ -11,3 +11,6 @@ details label#daemonStatusLabel {height: 10px; margin-top: 0; vertical-align: to
 details label.twoRowLabel {height: 2.0em;}
 
 form input[type=submit] {display: block; clear: both; margin: 0.5em auto 0 auto;}
+h1 {font-size: 24px; margin-bottom: 0.5em; font-family: sans-serif;}
+h1 img {height: 2.0em;}
+ul#uriList li {display: inline;}
```
Comment (ibnesayeed) [inline ipwb/assets/admin.css:16]: Do we really want to keep the list inline?

File: ipwb/replay.py [hunk 2]
```diff
@@ -575,12 +575,14 @@ def showAdmin():
                'htmlCount': htmlCount,
                'earliest': oldest_datetime,
                'latest': newest_datetime}
+
+    # TODO: determine why summary.htmlCount work propagate to template
     return render_template('admin.html', status=status, indexes=indexes,
-                           summary=summary)
+                           uris=uris, htmlCount=htmlCount, summary=summary)
```
Comment (ibnesayeed) [inline ipwb/replay.py:581]: Why are we supplying `uris` and `htmlCount` separately when they are already part of the `summary` object?

Comment (machawk1) [reply ipwb/replay.py:581]: @ibnesayeed See the comment just preceding 578. There is an issue of the values of summary, namely htmlCount, not being available in the template. This is a separate issue. For now, let's at least get the values displayed then refactor the implementation. :-)

File: ipwb/assets/admin.css (hunk 1 above)
Comment (machawk1) [reply ipwb/assets/admin.css:16]: @ibnesayeed I anticipate the design to be heavily refactored, e.g., #600. Inline here is to override the value pulled in from webui.css, which dictates that the values are initially hidden. We can revise this but (Step 1) Get the values display per the GitHub issue.

File: ipwb/replay.py (hunk 2 above)
Comment (machawk1) [reply ipwb/replay.py]: Resolved in 34882fe42bcd20d8a7b1292d9395fe6524559abd.

File: ipwb/assets/admin.css (hunk 1 above)
Comment (machawk1) [reply ipwb/assets/admin.css]: Resolved in 34882fe42bcd20d8a7b1292d9395fe6524559abd

Comment (ibnesayeed) [reply ipwb/assets/admin.css]: To undo the effect of an inherited CSS value in this case you could have used `display: list-item;` or `display: initial;`.

File: ipwb/replay.py [hunk 3]
```diff
@@ -575,12 +575,14 @@ def showAdmin():
                'htmlCount': htmlCount,
                'earliest': oldest_datetime,
                'latest': newest_datetime}
+
+    # TODO: determine why summary.htmlCount work propagate to template
```
Comment (ibnesayeed) [inline ipwb/replay.py:579]: Are you suggesting that other attributes of the `summary` object are available except a few?

Comment (machawk1) [reply ipwb/replay.py:579]: The effect is no longer occuring after removal, perhaps a caching issue.

**Comment by machawk1** [comment]
@ibnesayeed Approved on another medium. Merging!

**Pull request merged**